code/game/objects/structures/displaycase.dm

```diff
@@ -106,13 +106,25 @@
 	return
 
 /obj/structure/displaycase/attackby(obj/item/weapon/W, mob/user, params)
-	if(W.GetID() && electronics && !broken)
+	if(W.GetID() && !broken)

```

Comment:
No no, the captain displaycase being openable with captain ID is a good change, keep it.